requirements_for_test.txt

```diff
@@ -17,7 +17,7 @@ rfc3987==1.3.8
 # used for creating manifest file locally
 jinja2-cli[yaml]==0.6.0
 black==21.5b2
-locust==1.5.3
+locust==2.4.3
```

Comment:
@iokpala I hope that won't create compatibility issues and have bad repercussions. I will test with a local run before the merge.